src/renderers/dom/shared/SVGDOMPropertyConfig.js

```diff
@@ -263,16 +114,9 @@ var ATTRS = {
   xlinkTitle: 'xlink:title',
   xlinkType: 'xlink:type',
   xmlBase: 'xml:base',
-  xmlns: 0,
   xmlnsXlink: 'xmlns:xlink',
   xmlLang: 'xml:lang',
   xmlSpace: 'xml:space',
-  y: 0,
-  y1: 0,
-  y2: 0,
-  yChannelSelector: 'yChannelSelector',
-  z: 0,
-  zoomAndPan: 'zoomAndPan',
 };
 
 var SVGDOMPropertyConfig = {
```

Comment:
I feel like there's another step here where we could string manipulate `xml:lang` or `vert-origin-y` into camel-case, that might cut the overall gzip size.